src/c++/perf_analyzer/main.cc

```diff
@@ -1053,6 +1154,111 @@ main(int argc, char** argv)
         }
         break;
       }
+      case 31: {
+        std::string arg = optarg;
+        std::transform(
+            arg.begin(), arg.end(), arg.begin(),
+            [](unsigned char c) { return std::tolower(c); });
+        if (arg == "true") {
+          ssl_options.ssl_grpc_use_ssl = true;
+        } else if (arg == "false") {
+          ssl_options.ssl_grpc_use_ssl = false;
+        } else {
```

Comment:
Can you make it true if it exists, and false if it doesn't exist? Similar to the `--async` flag

Reply:
@debermudez for visibility

Elias and I had a lengthy discussion about this. Essentially, when MA calls PA, it must have values for each PA option. I don't know how to allow a PA option to both accept and not accept a value, so I've decided to make this option require a value, unfortunately.

@debermudez do you think you have time to modify the MA code so that it can work with PA options that don't require values?

Reply:
How about adding `None` as the value of the flags that don't have any value? If a flag needs to be only present, the user needs to provide `None` as the option value. I don't think we should restrict PA on what types of flags it can specify.

Reply:
After talking to @tanmayv25, I thought these were longs and not bools?  
I am passing values of type long to the PA client. This work shouldnt be necessary.

Reply:
Whoops, sorry Elias. There is _yet another_ new option we need to add. Check here for explanation:

https://github.com/triton-inference-server/server/pull/3874#discussion_r796240687

This thread is about the new option added, which is truly a bool.

Reply:
After our conversation during team meeting, I'm happy to keep this option here... I'll let @Tabrizian resolve this issue

Reply:
Yes, currently it will pass None but I was saying that we could change the parsing in MA for perf analyzer flags such that None means just passing the argument and not using the value.

Reply:
I think it would make the most sense for MA yaml user to work like this:
```
perf_analyzer_flags:
    some-other-option: /some/path
    ssl-grpc-use-ssl: false
```
⬇️
```
perf_analyzer --some-other-option=/some/path
```

```
perf_analyzer_flags:
    some-other-option: /some/path
    ssl-grpc-use-ssl: true
```
⬇️
```
perf_analyzer --some-other-option=/some/path --ssl-grpc-use-ssl
```

Reply:
Sounds good to me. I resolved this thread.